### PR TITLE
[FIX][IMP] web, website, mass_mailing: various fixes for milk

### PR DESCRIPTION
**Pull request by edi-odoo**
This PR fixes the following issues: 

- Embedded kanbans have a background mismatch  https://tinyurl.com/27yzdmc9
- We should probably align the icons for saving/discarding with the "cog" icon (since the goal was to locate them next to the record name, it works fine from a UX pov)  https://tinyurl.com/243un82c
- PDE: website "edit in backend" broken (product, blog, ...) https://bit.ly/3ng0dE7 
- PDE: website "edit" button not readable. https://bit.ly/3AKIS9j
- LNA: Those labels became really hard to read (too light) https://tinyurl.com/2mflz9zb

task-2818586

### CODE REVIEW

File: addons/mass_mailing/views/mailing_list_views.xml [hunk 1]
```diff
@@ -225,7 +225,7 @@
                                         <field name="contact_count_email"/>
                                     </span>
                                     <br/>
-                                    <span class="text-secondary">
```
Comment (stefanorigano) [inline addons/mass_mailing/views/mailing_list_views.xml:228]: Could you squash all "text-secondary -> text-muted" changes with the "Milk colors" commit?

**Review by stefanorigano — COMMENTED** [review]
Thanks @edi-odoo , very good 👍 
Just one remark

File: addons/mass_mailing/views/mailing_list_views.xml (hunk 1 above)
Comment (edi-odoo) [reply addons/mass_mailing/views/mailing_list_views.xml:228]: @stefanorigano Done!

**Comment by stefanorigano** [comment]
Thanks @edi-odoo 👍

**Pull request merged**